requirements.txt

```diff
@@ -3,4 +3,5 @@ numpy
 pandas
 pytest
 scipy
-flake8
\ No newline at end of file
+flake8
+sklearn
```

Comment:
These are test requirements, right?
For our package to install we shouldn't require sklearn. Does setup.py reflect that?

Reply:
Yes, those are just for test. Not to mention pytest

Reply:
Can we put a separate requirements.txt in the test directory, or does pip not work that way?

Reply:
I'm not 100% sure on whether this is right. I feel like there should be 1 place for package dependencies, e.g. setup.py, and another with the additional ones, but it doesn't seem to work like that.... Either way, let's just proceed and tackle this whenever we figure out the right way.